webpack.config.dev.js

```diff
@@ -6,7 +6,11 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const path = require('path');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 
-const noVisualization = process.env.ANALYSE_PACK.toString() === 'false';
+if (process.env.ANALYSE_PACK) {
```

Comment:
please made the same changes at webpack.config.prod.js and test the deploy at a heroku. 
also we use yarn as a package manager. so you add a new module, then you need to use the command yarn at console to generate a new yarn.lock file